pythonkr_backend/pythonkr_backend/celery.py

```
@@ -0,0 +1,41 @@
+@worker_init.connect()  
+def init_worker(*args, **kwargs):
+    logfire.configure(service_name="celery-worker")  
+    logfire.instrument_celery()
+
+@beat_init.connect()  
+def init_beat(*args, **kwargs):
+    logfire.configure(service_name="celery-beat")  
+    logfire.instrument_celery()
+
+@app.task
+def add(x: int, y: int):
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)